prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml

```diff
@@ -22,9 +23,88 @@ presubmits:
\ No newline at end of file
+        - ./prow/gcpproxy-presubmit.sh
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/cloudesf-testing-github-prow-service-account/service-account.json
+        - name: CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE
+          value: /etc/cloudesf-testing-github-prow-service-account/service-account.json
+        volumeMounts:
+        - name: cloudesf-testing-github-prow-service-account
+          mountPath: /etc/cloudesf-testing-github-prow-service-account
+        volumes:
+        - name: cloudesf-testing-github-prow-service-account
+          secret:
+            secretName: cloudesf-testing-github-prow-service-account
+  - name: ESPv2-gke-e2e-tight-http-bookstore-managed
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - --cluster=
+        - --deployment=gke
+        - --extract=gke-default
+        - --gcp-project=cloudesf-testing
+        - --gcp-zone=us-west1-b
+        - --gcp-network=default
+        - --gcp-node-image=gci
+        - --gke-create-command=container clusters create --cluster-ipv4-cidr=/19 --quiet
+        - '--gke-shape={"default":{"Nodes":1,"MachineType":"n1-standard-2"}}'
+        - --gke-environment=prod
+        - --provider=gke
+        - --env=TEST_CASE=tight-http-bookstore-managed
+        - --test=false
+        - --test-cmd=../prow/gcpproxy-e2e.sh
+        - --test-cmd-name=gcpproxy_e2e
+        - --timeout=80m
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20190509-e418529-master
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/cloudesf-testing-github-prow-service-account/service-account.json
+        - name: CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE
+          value: /etc/cloudesf-testing-github-prow-service-account/service-account.json
+        volumeMounts:
+        - name: cloudesf-testing-github-prow-service-account
+          mountPath: /etc/cloudesf-testing-github-prow-service-account
+        volumes:
+        - name: cloudesf-testing-github-prow-service-account
+          secret:
+            secretName: cloudesf-testing-github-prow-service-account
+  - name: ESPv2-cloud-run-e2e-cloud-run-http-bookstore
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/cloudesf-testing/gcpproxy-prow:v20191120-v0.0.0-125-geee50dd-master
+        env:
+        - name: TEST_CASE
+          value: cloud-run-cloud-run-http-bookstore
+        command:
+        - ./prow/gcpproxy-e2e.sh
+
+periodics:
+- name: ESPv2-continuous-build
+  cron: '0 * * * *' # Run by every hour
+  decorate: true
+  extra_refs:
+  - org: GoogleCloudPlatform
+    repo: esp-v2
+    base_ref: master
+  annotations:
+    testgrid-dashboards: googleoss-test-infra
```

Comment:
This dashboard is limited to jobs that maintain the testing infrastructure of this instance.

Please create a new dashboard for your team [here](https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/testgrid/config.yaml), such as `googleoss-espv2` or similar.